runtime/hsa-runtime/core/driver/xdna/amd_xdna_driver.cpp

```diff
@@ -117,7 +117,8 @@ hsa_status_t XdnaDriver::GetAgentProperties(core::Agent &agent) const {
     return HSA_STATUS_ERROR;
   }
 
-  aie_agent.SetNumCols(aie_metadata.cols);
+  // Right now can only target N-1 columns
```

Comment:
We may need some additional context here on why we can only target N-1 columns (e.g., current hardware limitations, something else).

Reply:
Good point will add more of an explanation.